apps/android/app/src/main/java/chat/simplex/app/views/usersettings/NetworkAndServers.kt

```diff
@@ -96,6 +102,31 @@ fun NetworkAndServersView(
           }
         }
       }
+    },
+    updateSessionMode = {
+      if (sessionMode.value == it) return@NetworkAndServersLayout
+      val prevValue = sessionMode.value
+      sessionMode.value = it
+      val startsWith = when (it) {
+        TransportSessionMode.User -> generalGetString(R.string.network_session_mode_user_description)
+        TransportSessionMode.Entity -> generalGetString(R.string.network_session_mode_entity_description)
+      }
+      updateNetworkSettingsDialog(
+        title = generalGetString(R.string.update_network_session_mode_question),
+        startsWith,
+        onDismiss = {  sessionMode.value = prevValue }
```

Comment:
probably didn't run formatter
```suggestion
        onDismiss = { sessionMode.value = prevValue }
```